snap-build/snap.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Setup the environment and build the snap image.
+# This script runs in the VM.
+
+set -x -e
+
+sudo apt-get update -y
+sudo apt-get install -y \
+	 snapd \
```

Comment:
Nit: Might be clearer to sort this list.

Reply:
yep, I agree